runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -4499,13 +4499,28 @@ TR::Register * J9::Power::TreeEvaluator::VMgenCoreInstanceofEvaluator(TR::Node *
 
    if (testCastClassIsSuper)
       {
-      TR::StaticSymbol *castClassSym;
+      bool depthInReg2 = false;
 
-      castClassSym = castClassSymRef->getSymbol()->getStaticSymbol();
-      void * clazz = castClassSym->getStaticAddress();
-      castClassDepth = TR::Compiler->cls.classDepthOf((TR_OpaqueClassBlock *) clazz);
+      TR::StaticSymbol    *castClassSym = NULL;
+      if (castClassSymRef && !castClassSymRef->isUnresolved())
+         castClassSym= castClassSymRef ? castClassSymRef->getSymbol()->getStaticSymbol() : NULL;
```

Comment:
The `classCastSymRef ? ...` test isn't needed since `if (classCastSymRef ...)` already tests it.

Reply:
Solved